.github/workflows/release.yml

```diff
@@ -41,14 +41,14 @@ on:
 
 jobs:
   release_core_bundle:
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
Any reason we use `macos-13` instead of `macos-latest`? Does latest not point to this version?

Reply:
"Does latest not point to this version?"

Yes. I have to double check if its still the case though.